src/ImportDefinitionsBundle/Form/Type/Interpreter/CoreShop/StoresInterpreterType.php

```diff
@@ -23,25 +23,25 @@
 
 final class StoresInterpreterType extends AbstractType
 {
+    /**
+     * {@inheritdoc}
+     */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
             ->add('stores', StoreChoiceType::class, [
                 'multiple' => true,
             ])
-            ->addModelTransformer(new CallbackTransformer
-            (
+            ->addModelTransformer(new CallbackTransformer(
                 function ($value) {
                     return $value;
                 },
                 function ($value) {
                     $resolvedValues = [];
-                    if (is_array($value) && array_key_exists('stores', $value)) {
-                        if ($value['stores'] instanceof ArrayCollection) {
-                            foreach ($value['stores'] as $val) {
-                                if ($val instanceof StoreInterface) {
-                                    $resolvedValues[] = $val->getId();
-                                }
+                    if (\is_array($value) && array_key_exists('stores', $value) && $value['stores'] instanceof ArrayCollection) {
```

Comment:
not quite pretty though. maybe something like

```
if (!is_array($value)  {
return;
}
if (!array_key_exists('stores', $value)) {
return;
}
....
```

Reply:
What about...

```php
if (!\is_array($value) ||
    !array_key_exists('stores', $value) ||
    !$value['stores'] instanceof ArrayCollection) {
    return [];
}
    
foreach ($value['stores'] as $val) {
    if ($val instanceof StoreInterface) {
        $resolvedValues[] = $val->getId();
    }
}
```

?

Reply:
👍